truck_inspection_client/src/truck_inspection_client.cpp

```diff
@@ -84,7 +84,8 @@ namespace truck_inspection_client
             mo_msg.strategy = this->INSPECTION_STRATEGY;
             std::string ads_auto_status = this->ads_engaged_ ? "Engaged" : "Not Engaged";
             std::string ads_health_status = ads_system_alert_type_ ;
-            std::string params = boost::str(boost::format("vin_number:%s,license_plate:%s,carrier_name:%s,carrier_id:%s,weight:%d,ads_software_version:%s,date_of_last_state_inspection:%s,date_of_last_ads_calibration:%s,pre_trip_ads_health_check:%s,ads_health_status:%s,ads_auto_status:%s,iss_score:%d,permit_required:%s")
+            //std::string params = boost::str(boost::format("vin_number:%s,license_plate:%s,carrier_name:%s,carrier_id:%s,weight:%d,ads_software_version:%s,date_of_last_state_inspection:%s,date_of_last_ads_calibration:%s,pre_trip_ads_health_check:%s,ads_health_status:%s,ads_auto_status:%s,iss_score:%d,permit_required:%s")
+            std::string params = boost::str(boost::format("v:%s,l:%s,n:%s,c:%s,w:%d,o:%s,f:%s,a:%s,r:%s,h:%s,u:%s,i:%d,p:%s")
```

Comment:
I thought we agreed to just do in alphabetical order? (e.g. a:%s,b:%s,c:%s, and so on). I think there's no meaning to the field names anymore that following first character doesn't say much on the other end. 

If you'd like to use what you have now, what letter logic did you choose from weight to iss_score (o:%s,f:%s,a:%s,r:%s,h:%s,u:%s,)?